client/ayon_core/pipeline/create/context.py

```diff
@@ -2043,7 +2043,8 @@ def create(
         variant,
         folder_entity=None,
         task_entity=None,
-        pre_create_data=None
+        pre_create_data=None,
+        active=True
```

Comment:
I don't like that `create` method changed signature. Shoulnd't that be part of `pre_create_data`? Or deactivate the instance after creation?

Reply:
Done with https://github.com/ynput/ayon-core/pull/774/commits/849ab8b624586db321b3615e5ee6e71aa94b54df